release/ray_release/scripts/build_pipeline.py

```diff
@@ -237,5 +240,14 @@ def main(
     print(steps_str)
 
 
+def _build_anyscale_byod_images(tests: List[Tuple[Test, bool]]) -> None:
+    """
+    Builds the Anyscale BYOD images for the given tests.
+    """
+    _ = {test.get_ray_image() for test, _ in tests}
+    # TODO(aslonnie): Build anyscale byod images for the given ray images
+    return
```

Comment:
Suggestions: Let's move this to a separate package e.g. `ray_release.byod` which also contains the dockerfile, dependencies, etc

Reply:
I like that!